mleap-spark/src/main/scala/org/apache/spark/ml/bundle/ops/regression/GeneralizedLinearRegressionOp.scala

```diff
@@ -42,9 +42,12 @@ class GeneralizedLinearRegressionOp extends SimpleSparkOp[GeneralizedLinearRegre
   }
 
   override def sparkLoad(uid: String, shape: NodeShape, model: GeneralizedLinearRegressionModel): GeneralizedLinearRegressionModel = {
-    new GeneralizedLinearRegressionModel(uid = uid,
+    val m = new GeneralizedLinearRegressionModel(uid = uid,
       coefficients = model.coefficients,
       intercept = model.intercept)
+    m.set(m.family, model.getFamily)
+    m.set(m.link, model.getLink)
```

Comment:
I think that this may or may not be serialized, if you look at the store() method above, we need to do something similar to the load() method above.